core/modules/query/TableRef.h

```diff
@@ -111,9 +111,9 @@ class TableRef {
 class TableRef::render {
 public:
     render(QueryTemplate& qt) : _qt(qt), _count(0) {}
-    void operator()(TableRef const& trn);
-    inline void operator()(TableRef::Ptr const trn) {
-        if(trn.get()) (*this)(*trn);
+    void apply(TableRef const& trn);
+    inline void applyToQT(TableRef::Ptr const trn) {
```

Comment:
Just missed changing apply's name. Just trying to make it clearer that it's modifying the underlying QueryTemplate object.